### PR TITLE
Jimmy float syntax 

### PR DESCRIPTION
**Pull request by rxie25**
fixed the float problem. Now accepts 42, 42.0 and 42. It past the tests merge it if you can.

### CODE REVIEW

**Comment by jzacsh** [comment]
I see the Python file already got into master. Does that mean this whole pull request has already been manually side-merged?

File: src/scanner.mll [hunk 1]
```diff
@@ -34,7 +34,7 @@ rule token = parse
 | "while"  { WHILE }
 | "find"   { FIND }
 | '"' (([^'"']*) as lxm) '"' { STRLIT(lxm) }
-| ['0'-'9']+ as lxm { LITERAL(int_of_string lxm) }
+| ['0'-'9']+|['0'-'9']+['.']['0'-'9']* as lxm { LITERAL(float_of_string lxm) }
```
Comment (jzacsh) [inline src/scanner.mll]: Our LRM - here: https://github.com/jzacsh/PLT2016Spring/blob/master/notes/language-reference-manual.md#tokens-expressions-lexemes - explicitly outlines something closer to the float regexp we had on HW1. Can you do that here?

File: src/tests/test-print42double.eq [hunk 2]
```diff
@@ -0,0 +1,7 @@
+SomeContext = {
+  a = { 42.; }
```
Comment (jzacsh) [inline src/tests/test-print42double.eq]: (related to the homework comment above) can you add more test cases for all the edge cases we've learned about?

File: src/tests/test-print42double.eq [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+}
+
+SomeContext:find a {
+  print("%.0f\n", a);
```
Comment (jzacsh) [inline src/tests/test-print42double.eq]: Casting away the floating point with printf formatter really masks the success of the test here :P

File: src/scanner.mll (hunk 1 above)
Comment (rxie25) [inline src/scanner.mll]: I think Lanting has pull requested her version of int/float to the master. So we can just delete this pull request.

File: src/tests/test-print42double.eq (hunk 3 above)
Comment (rxie25) [inline src/tests/test-print42double.eq]: yeah, need to change that test to show float.

**Comment by jzacsh** [comment]
continuing in #40 since this got abandonned
